packages/flow-parser-bin/package.json

```diff
@@ -31,6 +31,7 @@
     "linux"
   ],
   "cpu": [
-    "x64"
+    "x64",
+    "arm64"
```

Comment:
Please revert this. You can do it in another PR, where you also update the CI pipeline that actually include the arm64 artifact.